browser/src/control/Control.ContextToolbar.ts

```diff
@@ -282,6 +290,38 @@ class ContextToolbar {
 		];
 	}
 
+	insertAdditionalContextButton(button: any) {
+		for (const i in this.additionalContextButtons) {
+			const item = this.additionalContextButtons[i];
+			if (item.id === button.id) return;
+		}
+
+		if (this.additionalContextButtons.length === 0) {
+			this.additionalContextButtons.push({
+				type: 'separator',
+				id: 'home-insertannotation-break',
+				orientation: 'vertical',
+			} as SeparatorWidgetJSON);
+		}
+
+		const isUnoCommand =
+			button.unoCommand && button.unoCommand.indexOf('.uno:') >= 0;
+		if (button.unoCommand && !isUnoCommand)
+			button.unoCommand = '.uno:' + button.unoCommand;
+
+		const contextButton = {
+			id: button.id,
+			type: button.type ? button.type : 'toolitem',
+			text: button.label ? button.label : button.hint ? _(button.hint) : ' ',
+			command: button.unoCommand,
```

Comment:
Needs a postMessage callback when unoCommand was not passed.

Reply:
With `postmessage`, now Control.Menubar's _executeAction will fire 'Clicked_Button'.
Is it correct ?

Would be nice of you to add a reply, since you changed the code.

Reply:
Now 'Clicked_ContextualButton' will be fired.

Reply:
Well I was fine having a single event 'Clicked_Button' since it had an id parameter to distinguish buttons already.
Having a separate one is ok as well.